### ISSUE TITLE
Clarify the relationship between the two packages, and general usage

### ISSUE THREAD

**Issue by roji** [dotnet/reproducible-builds#15]
Reading the [README.md](https://github.com/dotnet/reproducible-builds), it isn't quite clear whether one should use both DotNet.ReproducibleBuilds and DotNet.ReproducibleBuilds.Isolated, or only one of them. What exactly each one of these packages does is also a bit vague, i.e. what exactly do I opt into for each one.

One last note: for DotNet.ReproducibleBuilds.Isolated, the usage says to include an `<Sdk ... />` element in the csproj or Directory.Builds.props. A more complete sample would be useful here, since Sdk elements aren't something that's typically used, so it may not be immediately clear where to place them (inside an <ItemGroup>? outside?).

**Comment by cmeeren** [comment]
Can someone explain how to use the `<Sdk>` element in the Directory.Build.props? I can't find any information about where to add this line from the readme:

```<Sdk Name="DotNet.ReproducibleBuilds.Isolated" Version="{TBD}" />```

**Comment by baronfel** [comment]
Hey folks!

There are some docs on how to reference SDKs in general on the MSBuild Docs site: https://docs.microsoft.com/en-us/visualstudio/msbuild/how-to-use-project-sdk?view=vs-2022

So I'd expect to see a project file or Directory.Build.props like:

```
<Project>
  <Sdk Name="Dotnet.ReproducibleBuilds.Isolated" Version="111" />
</Project>
```
@cmeeren for the specific build error, do you have a sample I can try to repro with locally?

Finally, I don't see anything in the props/targets files for the two packages that would prevent their use together. I'd need to dig deeper to be sure, though.

**Comment by cmeeren** [comment]
@baronfel You're right, my error was caused by Rider unloading the projects (without my noticing) when I made an erroneous change to `Directory.Build.props`, and there were then no projects to actually build.

It works with

```xml
<Project>

  <Sdk Name="DotNet.ReproducibleBuilds.Isolated" Version="1.1.1" />

  ...
```

**Comment by hcoona** [comment]
Checked the source code. I think

* `DotNet.ReproducibleBuilds` is only for introduce the SourceLink & configure it properly for mainline CI systems.
* `DotNet.ReproducibleBuilds.Isolated` is only for applying the suggested configurations recorded in https://github.com/dotnet/reproducible-builds/blob/main/Documentation/Reproducible-MSBuild/Techniques/toc.md
